docs/source/release.rst

```diff
@@ -8,6 +8,7 @@ Release Notes
    notes for pre-1.0 versions of ibis can be found at :doc:`/release-pre-1.0`
 
 * :release:`1.2.1 <pending>`
+* :bug:`2030` Fix LinuxBuildDocs issue related to semantic_version, fix doc section inside Makefile and skip mysql tzinfo on CI
```

Comment:
from a user point of view this is not very useful. what IS useful and worth mentioning, is that we now have a pin in the docs build (its not really user, but developer user here)

Reply:
sounds good. I updated the release note. let me know if it needs any other change.